internal/installcfg/installcfg.go

```diff
@@ -180,6 +180,9 @@ func setBMPlatformInstallconfig(log logrus.FieldLogger, cluster *common.Cluster,
 	sortedHosts := make([]*models.Host, len(cluster.Hosts))
 	copy(sortedHosts, cluster.Hosts)
 	sort.Slice(sortedHosts, func(i, j int) bool {
+		if sortedHosts[i].Role != sortedHosts[j].Role {
```

Comment:
Why this logic?

Reply:
so that the masters will be the first hosts defined, before workers

Reply:
Consider adding a comment about the logic:
// Sorting hosts by by hostname, but master takes precedence before worker.